### PR TITLE
Updates to vSphere CPI documentation

### PR DESCRIPTION
**Pull request by nehagjain15**
Add documentation for Managed_networks and VM Groups.

### CODE REVIEW

File: content/vsphere-cpi.md [hunk 1]
```diff
@@ -32,6 +32,19 @@ Schema for `cloud_properties` section used by manual network subnet:
 
 * **name** [String, required]: Name of the vSphere network. Example: `VM Network`.
 
+### Managed Networks ###
+Create T1 router and attach it to T0 router. It also creates virtual switch and attaches it to T1 router.
+So every subnet in network will have its own switch. Route advertisement is enabled on T1 router.
```
Comment (mfine30) [inline content/vsphere-cpi.md]: Is this paragraph intended to describe steps that the operator should take or to explain what the Managed Networks feature does? i.e. my recollection is that the T0 router is a prerequisite, but BOSH will create the T1 router.

File: content/vsphere-cpi.md [hunk 2]
```diff
@@ -32,6 +32,19 @@ Schema for `cloud_properties` section used by manual network subnet:
...
+
+Schema for `cloud_properties` section used by managed network subnet:
+
+* **name** [String, required]: Name of the vSphere network. Example: `Subnet 1`.
```
Comment (mfine30) [inline content/vsphere-cpi.md]: Is this a logical name used inside of a deployment manifest; if so should it be something more akin to `subnet_1` or something without spaces/odd characters.

File: content/vsphere-cpi.md (hunk 1 above)
Comment (nehagjain15) [reply content/vsphere-cpi.md]: This is to explain what managed networks does. Should I instead update it with following description? And you are right T0 router is a prerequisite

"Automates creation of network on NSX-T. New network will have set of T1 Router and virtual switch"

File: content/vsphere-cpi.md (hunk 2 above)
Comment (nehagjain15) [reply content/vsphere-cpi.md]: Will update it

File: content/vsphere-cpi.md (hunk 1 above)
Comment (mfine30) [reply content/vsphere-cpi.md]: I would rephrase it to "Creates a T1 router and attaches it to T0 router. It will also create a virtual switch and attach it to the T1 router"

**Comment by nehagjain15** [comment]
@mfine30 I have updated the docs

**Comment by mfine30** [comment]
Merged! Thanks!
